tests/IntegrationTest.js

```diff
@@ -7,19 +7,19 @@ const root = path.join(__dirname, '..');
 
 describe('Integration', () => {
   it('deploys the contracts, starts reporters and posts the right prices', async () => {
-  await compose.upAll({cwd: root, log: true});
-  try {
-    await new Promise(ok => setTimeout(ok, 3000));
+    try {
+      await compose.upOne(["poster"], {cwd: root, log: true});
```

Comment:
Was this for debugging?

Reply:
kind of works better this way since the poster has all the required dependencies